src/Uno.UI/UI/Xaml/Data/Binding.cs

```diff
@@ -217,6 +216,9 @@ public object CompiledSource
 		/// </summary>
 		internal string[] XBindPropertyPaths { get; private set; }
 
+		// Each of these values could be null and the Binding could still be an x:Bind, but they can't all be null
+		public bool IsXBind => XBindSelector is not null || XBindPropertyPaths is not null || CompiledSource is not null || XBindBack is not null;
```

Comment:
This shouldn't be public

Reply:
Let's make it an extension on markuphelper